lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -44,43 +57,82 @@ local function playerIsInBuild( ply )
     return !getPlayerPvpMode( ply )
 end
 
-local function movementMultiplier(weaponCount) 
-    return math.Clamp(1 - (1.9^( weaponCount))/100, 0, 1)
+local minMult = 0.2
+local function movementMultiplier( weaponCount )
+    local N = 1 - (1.9^( weaponCount ))/100
```

Comment:
A comment explaining the math would be helpful.

Better yet, assign each of these values a descriptive variable name (and split the math into multiple steps) so when they're combined the reader understands.